src/imguiclient/imguiapp.cpp

```diff
@@ -1410,66 +1404,77 @@ std::string ImGuiApp::create_time_string(std::chrono::time_point<std::chrono::sy
+    ImGui::PopStyleVar(3);
+
+    // Submit the DockSpace
+    ImGuiIO &io = ImGui::GetIO();
+    if (io.ConfigFlags & ImGuiConfigFlags_DockingEnable)
     {
-        ImGui::SetWindowPos("main", m_windowPos);
-        ImGui::SetWindowSize("main", ImVec2(m_windowSize.x, 0.f));
+        ImGuiID dockspace_id = ImGui::GetID("MyDockSpace");
+        ImGui::DockSpace(dockspace_id, ImVec2(0.0f, 0.0f), ImGuiDockNodeFlags_PassthruCentralNode);
+    }
 
-        ImScoped::MenuBar menu_bar;
-        if (menu_bar.IsOpen)
+    // Menu bar code
+    if (ImGui::BeginMenuBar())
+    {
+        if (ImGui::BeginMenu("File"))
         {
+            if (ImGui::MenuItem("Exit"))
             {
-                ImScoped::Menu menu("File");
-                if (menu.IsOpen)
-                {
-                    if (ImGui::MenuItem("Exit"))
-                    {
-                        // Will wait for all work to finish and then shutdown the app.
-                        prepare_shutdown_nowait();
-                    }
-                    ImGui::SameLine();
-                    TooltipTextUnformattedMarker("Graceful shutdown. Finishes all tasks before exiting.");
-                }
+                prepare_shutdown_nowait();
             }
+            ImGui::SameLine();
+            TooltipTextUnformattedMarker("Graceful shutdown. Finishes all tasks before exiting.");
+            ImGui::EndMenu();
+        }
 
-            {
-                ImScoped::Menu menu("Tools");
-                if (menu.IsOpen)
-                {
-                    ImGui::MenuItem("Program File Manager", nullptr, &m_showFileManager);
-                    ImGui::MenuItem("Assembler Output", nullptr, &m_showOutputManager);
+        if (ImGui::BeginMenu("Tools"))
+        {
+            ImGui::MenuItem("Program File Manager", nullptr, &m_showFileManager);
+            ImGui::MenuItem("Assembler Output", nullptr, &m_showOutputManager);
 
-                    if (ImGui::MenuItem("New Assembler Comparison"))
-                    {
-                        add_program_comparison();
-                    }
-                    ImGui::SameLine();
-                    TooltipTextUnformattedMarker("Opens a new Assembler Comparison window.");
-                }
+            if (ImGui::MenuItem("New Assembler Comparison"))
+            {
+                add_program_comparison();
             }
+            ImGui::SameLine();
+            TooltipTextUnformattedMarker("Opens a new Assembler Comparison window.");
+            ImGui::EndMenu();
         }
+        ImGui::EndMenuBar();
     }
+
+    ImGui::End();
 }
 
 void ImGuiApp::FileManagerWindow(bool *p_open)
 {
-    ImGui::SetNextWindowSizeConstraints(ImVec2(400, 300), ImVec2(FLT_MAX, FLT_MAX));
-
-    ImScoped::Window window("File Manager", p_open, ImGuiWindowFlags_MenuBar);
+    ImGuiWindowFlags window_flags = ImGuiWindowFlags_MenuBar;
+    ImScoped::Window window("File Manager", p_open, window_flags);
```

Comment:
Why was this code changed? Was the merge conflict incorrectly resolved?

Reply:
Most likely yes, I tried choosing your branch at each conflict, but I may have missed this one

Reply:
Fixed.